roles/lifecycle_environments/README.md

```diff
@@ -0,0 +1,64 @@
+theforeman.foreman.lifecycle_environments
+=========================================
+
+This role creates and manages Lifecycle Environments.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+The main data structure for this role is the list of `lifecycle_environments`. Each `lifecycle_environment` requires the following fields:
+
+- `name`: The name of the lifecycle environment.
+- `prior`: The name of the previous lifecycle environment to attach to in sequence
```

Comment:
```suggestion
- `prior`: The name of the previous lifecycle environment to attach to in sequence. For the first lifecycle environment in a new path, set the prior lifecycle environment to Library.
```

This will likely be helpful to somebody.